.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,99 @@
+name: Bug Report
+description: File a bug report.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: "**Required Info:**"
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    id: regression
+    attributes:
+      label: Regression
+      description: Is the reported bug a regression? If so, what is the last version of ROS where it still worked fine?
+  - type: input
+    id: os
+    attributes:
+      label: "Operating System:"
+      description: |
+        Please try to be specific.
+        For Linux, please use the command `uname -a` from a terminal and copy paste its output here.
+        For Windows, open a terminal (Win key + R and type `cmd`), type the command `ver` and press enter.
+        Then copy paste the output here.
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: "ROS version or commit hash:"
+      description: |
+        **Examples:**  *humble*, *iron*, *jazzy*, ...
+    validations:
+      required: true
+  - type: input
+    id: rmw
+    attributes:
+      label: "RMW implementation:"
+      description: |
+        **Examples:** *rmw_fastrtps_cpp*, *rmw_connextdds*, *rmw_cyclonedds_cpp*, ...
+        You can check the ROS Middleware (RMW) implementation with the command: `ros2 doctor --report`
+        Find the line starting with `middleware name` in the report.
+    validations:
+      required: true

```

Comment:
maybe we can change this into `if applicable` to be general issue template for ros2 org, because some ros2 repositories do not need to have this information such as `ros2_documentation`.